### ISSUE TITLE
Fix accessibility of Select component

### ISSUE THREAD

**Issue by witoszekdev** [saleor/react-storefront#373]
Our current `<Select>` component isn't focusable by keyboard and is not visible to screen readers. It also doesn't support arrow navigation.

It should confront to the Listbox specification defined in WAI APG: https://www.w3.org/WAI/ARIA/apg/patterns/listbox/

**Comment by mmiszy** [comment]
Fixed in https://github.com/saleor/react-storefront/pull/477
